invesalius/data/elfin.py

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env python3
+
+import sys
+from time import sleep
+from socket import socket, AF_INET, SOCK_DGRAM, SOCK_STREAM
+import invesalius.constants as const
+
+class elfin_server():
+    def __init__(self, server_ip, port_number):
+        self.server_ip = server_ip
+        self.port_number = port_number
+
+    def Initialize(self):
```

Comment:
These functions would benefit from documentation. What is the SIZE and rbtID variables? Will they every change or they are the same for all Elfin? Could write this here as doc.

Replace `rbtID` by `robot_ID`